content/blog/2021-02-12-february-21-dvc-heartbeat.md

```diff
@@ -0,0 +1,143 @@
+﻿---
+title: February'21 Heartbeat
+date: 2021-02-12
+description: |
+  Monthly updates are here! read all about our growing team, 
+  our CEO's interview on The New Stack, integration with spaCy and more!
+descriptionLong: |
+  Monthly updates are here! read all about our growing team, 
+  our CEO's interview on The New Stack, integration with spaCy and more!
+picture: 2021-01-20/cover.png
+author: elle_obrien
+commentsUrl: https://discuss.dvc.org/t/dvc-heartbeat-jan-21/632
+tags:
+  - Heartbeat
+  - CML
+  - DVC
+  - DAGsHub
+  - spaCy
+  - ML Summit 2021
+  - Spell
+  - MLOps
+---
+
+## News
+
+Happy February! Here's all the news to keep you up to date!
+
+## We've hired and are still hiring!
+
+We have four new team members this month!
+
+[**Dave Berenbaum**](https://www.linkedin.com/in/david-berenbaum-20b6b424/) came
+to Iterative.ai by way of a previous contribution to our open source products
+while working as a Data Science Manager at Captial One. He joins the team as a
+Technical Product Manager. We are thrilled he's here!
+
+[**Batuhan Taskaya**](https://www.linkedin.com/in/batuhan-osman-taskaya-7803b61a0/)
+joins us as a DVC Software engineer working on the Python core. Batuhan is
+excited to work on open source full time and we are excited to have him do so!
+
+[**Jeny De Figueiredo**](https://www.linkedin.com/in/jenifer-de-figueiredo/) is
+involved in the Seattle area data science community at Data Circles and is a
+WiDS Puget Sound Ambassador. She joins us as our new Community Manager and is
+looking forward to further building and engaging the community in MLOps!
+
+[**Roger Parent**](https://www.linkedin.com/in/rogermparent/) has already been a
+big part of building DVC and CML. He has been a primary developer of a Visual
+Studio code extension that interfaces with the DVC Python application to provide
+an interface with the Experiments featurre that's coming out with DVC 2.0. We
+are so excited to have him joining us full time as
+
+![Alt Text](https://media.giphy.com/media/vAvWgk3NCFXTa/giphy.gif)
+
+We are on the hunt for a
+[Senior Software Engineer - Open Source, Dev Tools](https://weworkremotely.com/remote-jobs/iterative-senior-software-engineer-open-source-dev-tools-3).
```

Comment:
let's put this job listing please instead - this one is way more important now https://docs.google.com/document/d/1aT5HZYt4kAUxXqD4JNTe3jPDlVUwSmnEWDPR2QoKdvo/edit

Reply:
@shcheklein I don't see this posted at weworkremotely.  Are you wanting to just connect to the gdrive link?  (I think it would be better if it were posted.  It looks rinky dink coming from a google drive.). If we don't have time for that I could put 'hot off the presses" or something.

Reply:
we have it on PH - https://www.producthunt.com/jobs/iterative-sr-engineer-ts-node-react-open-source-dev-tools - but PH doesn't have a nice description

but also people do GDrive - check this org (link at the very bottom) https://mightyapp.com/ - they are well known and respected in the community.

Reply:
Ok.  If you're comfortable with that then I will do it. 👍🏼